components/brave_wallet_ui/common/slices/api.slice.ts

```diff
@@ -1521,8 +1458,8 @@ export function createWalletApi (
                 : (
                     await Promise.all(
                       (
-                        await cache.getWalletInfo()
-                      ).accountInfos.map(async (account) => {
+                        await cache.getAllAcountsInfo()
+                      ).accounts.map(async (account) => {
```

Comment:
`accounts` are already fetched/defined on line 1312
```suggestion
                        accounts
                      ).map(async (account) => {
```